Dockerfile.c7

```diff
@@ -16,19 +16,7 @@ RUN pip3 install git+git://github.com/pearu/pylibtiff
 COPY . /git/ome-files-py
 
 WORKDIR /git/ome-files-py
-RUN python2 setup.py install
-
-WORKDIR /git/ome-files-py/test
-RUN python2 all_tests.py
-RUN python2 ../examples/dump_planes.py data/multi-channel-4D-series.companion.ome
-
-WORKDIR /git/ome-files-py
-RUN git clean -dfx
-RUN python3 setup.py install
-
-WORKDIR /git/ome-files-py/test
-RUN python3 all_tests.py
-RUN python3 ../examples/dump_planes.py data/multi-channel-4D-series.companion.ome
+RUN python2 setup.py install && git clean -dfx && python3 setup.py install
```

Comment:
Can we use `TRAVIS_PYTHON_VERSION` here to build only once, for the python version we're testing?

Reply:
Isn't this variable only be restricted to the usage in Travis or how would the `Dockerfile` be generically modified? In general, building one Docker image per platform and per Python version is technically doable but we might discuss the advantages compared to the current situation where Python2 or Python3 can be selected at runtime.

Reply:
The Docker containers aren't just for testing, they also serve as the quickest way for a user to try the library. I think having it compiled for both Python versions is a plus.